keanu-python/tests/test_sampling.py

```diff
@@ -58,7 +62,20 @@ def test_down_sample_interval(net: BayesNet) -> None:
         net=net, sample_from=net.get_latent_vertices(), draws=draws, down_sample_interval=down_sample_interval)
 
     expected_num_samples = draws / down_sample_interval
-    assert all(len(vertex_samples) == expected_num_samples for vertex_id, vertex_samples in samples.items())
+    assert all(len(vertex_samples) == expected_num_samples for label, vertex_samples in samples.items())
+
+
+@pytest.mark.mpl_image_compare(filename='test_sample_with_plot.png')
+def test_sample_with_plot() -> Any:
+    with Model() as m:
+        m.exp = Exponential(np.ones((2, 2)))
+        m.exp.observe(np.array([[3., 2.], [0., 1.]]))
+        m.bernoulli = Bernoulli(m.exp)
+    net = m.to_bayes_net()
+
+    fig, ax = plt.subplots(1, 1, squeeze=False)
+    sample(net=net, sample_from=net.get_observed_vertices(), draws=5, plot=True, ax=ax)
```

Comment:
why is the graph constant? i.e. every sample is the same

Reply:
oops, forgot we could seed `KeanuRandom` in Python now.